docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
```

Comment:
Do we want this in the repo?

Reply:
I kind of do. since I had to look up the docker-compose spec to write it. docker-compose is still a good test for local development. I'm ok with taking it out also.